core/src/trezor/lvglui/scrs/homescreen.py

```
@@ -273,24 +272,25 @@
-            self.address_desc.add_style(
+            self.my_address = lv.imgbtn(self)
+            self.my_address.set_size(216, 216)
+            self.my_address.align_to(self.connect, lv.ALIGN.OUT_BOTTOM_MID, 0, 77)
+            self.my_address.set_style_bg_img_src("A:/res/app-address.jpg", 0)
+            self.my_address.add_style(click_style, lv.PART.MAIN | lv.STATE.PRESSED)
+            self.my_address.add_flag(lv.obj.FLAG.EVENT_BUBBLE)
+            self.my_address_desc = lv.label(self)
+            self.my_address_desc.set_text(_(i18n_keys.APP__ADDRESS))
+            self.my_address_desc.add_style(default_desc_style, 0)
+            self.my_address_desc.add_style(
                 pressed_desc_style, lv.PART.MAIN | lv.STATE.PRESSED
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)